setup/upgrade_major.rst

```diff
@@ -182,6 +182,16 @@ Next, use Composer to download new versions of the libraries:
 
     $ composer update "symfony/*"
 
+It's also recommended to clear the cache after updating to a new major version:
+
+.. code-block:: terminal
+
+    # this deletes the development cache in your local machine
+    $ php bin/console cache:clear
```

Comment:
Running the `cache:clear` command will not solve the situation described in the linked issue if I understand it correctly. Being able to run this command means that your console application needs to be able to boot which can lead to the described issues if the dumped container is not compatible with your vendor code. The only solution to fix it is to remove the cache directory from the filesystem and to warm up the cache afterwards.

Reply:
I've just reworded as you proposed. Thanks for the review!